todo_app/templates/index.html

```diff
@@ -12,14 +12,14 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         {% for item in items %}
-          <li>{{item}}</li>
+          <li>id: {{item.id}}          Status:  {{item.status}}          Name: {{item.name}}</li>
```

Comment:
If we do go with the above suggestion re tweaking the complete_item endpoint, we should be able to put in a simple button to connect these like:
```
          <li>id: {{item.id}} Status: {{item.status}} Name: {{item.name}}</li>
          <form action="/complete_item/{{item.id}}" method="POST"><input type="submit" value="Complete"/></form>
```